Makefile

```diff
@@ -213,7 +216,7 @@ endif
 BUNDLE_IMGS ?= $(BUNDLE_IMG)
 
 # The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
-CATALOG_IMG ?= $(IMAGE_TAG_BASE)-index:v$(VERSION)
+CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION)
```

Comment:
well the new default is catalog i can revert this but this was the name used by the SDK for a new repo.

so either we need to maintain index for ever or we can use the default value that is use in the SDK now before we have any users of this.

if we defer changing it and need to change it later it will be more work then changing it now but i don't really mind either way.